Meta/check-png-sizes.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+# How many bytes optipng has to be able to strip out of the file for the optimization to be worth it. The default is 2 KiB.
+MINIMUM_OPTIMIZATION_BYTES="${MINIMUM_OPTIMIZATION_BYTES:-1024}"
```

Comment:
Comment and commit message are incorrect

Reply:
Also, this is usually written as 
```bash
: "${MINIMUM_OPTIMIZATION_BYTES:=1024}"
```

Reply:
hrm, tell that to the other scripts I copied the pattern from, but will change

Reply:
```
[test@turingcomplete] ~/Workspace/serenity
❯ rg --pcre2 ': "\$\{\w+:=.*\}"' -c
Meta/build-image-limine.sh:1
Meta/build-image-qemu.sh:1
Meta/build-image-grub.sh:1
Meta/build-native-partition.sh:1
Meta/build-image-extlinux.sh:1
Meta/run.sh:1
                                                                                 
[test@turingcomplete] ~/Workspace/serenity
❯ rg --pcre2 '(\w+)="\$\{\1:-.*\}"' -c
Meta/build-root-filesystem.sh:2
Meta/run.sh:9
Kernel/embedmap.sh:1
Kernel/mkmap.sh:1
```

Looks like the one you copied is more common, feel free to leave it as-is.